XiEditor/LineCache.swift

```diff
@@ -92,7 +82,7 @@ class LineCache {
         }
         return nil
     }
-
+    
```

Comment:
Yea, Xcode seems to have a bias towards ending files with a newline. I'm confused by this diff though, since my local revision has no trailing newline and is up to date. Github has occasional bugs around rendering whitespace in diffs, not sure if this is one of those instances?